src/apps/projinfo.cpp

```diff
@@ -1042,8 +1043,7 @@ int main(int argc, char **argv) {
         usage();
     }
 
-    std::string user_string;
-    bool user_string_specified = false;
+    std::vector<std::string> user_strings;
```

Comment:
perhaps ``positional_args`` would be clearer ?

Reply:
done